openedx/core/djangoapps/course_apps/toggles.py

```diff
@@ -19,9 +19,28 @@
     f'{COURSE_APPS_WAFFLE_NAMESPACE}.proctoring_settings_modal_view', __name__
 )
 
+# .. toggle_name: course_apps.exams_ida_enabled
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Slowly replace edx-proctoring api calls with calls to edx-exams
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2022-07-27
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: MST-1520
+EXAMS_IDA_OVERIDE = CourseWaffleFlag(
```

Comment:
naming nit - if the flag is called exams_ida_enabled might as well call the constant EXAMS_IDA_ENABLED

also enabled is a bit weird in a waffle flag because enabled/not enabled is why you have the flag. EXAMS_IDA might be enough.

Reply:
EXAMS_IDA is probably enough. I think override might be the wrong word because it implies we're totally replacing edx-proctoring which isn't entirely true

Reply:
Ah got it, this makes sense